layers/org/packages.el

```diff
@@ -290,6 +290,21 @@ Will work on both org-mode and any mode that accepts plain html."
       (spacemacs/set-leader-keys
         "Cc" 'org-capture)
 
+      (unless (version< emacs-version "24.4")
+
+        (defun spacemacs/org-clearer-view ()
```

Comment:
@syl20bnr Because people will think I'm being serious. Am I? I don't even know :confused: 
I should reduce the amount of :hankey:  in it before giving diz `smelly-foo` any proper name